PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+
+# What you've done
+...
+
+# Why have you done it
+...
+
+# Screenshot (if visual)
+...
+
+# Any testing carried out
+...
```

Comment:
Yeah, but I think we shouldn't have it on the template. That's up to the person opening the PR. We can, on the other hand, add a link to the contribution guidelines.